src/Tools/ExternalAccess/Razor/RazorPredefinedCodeFixProviderNames.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CodeFixes;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Razor
+{
+    internal static class RazorPredefinedCodeFixProviderNames
+    {
+        public const string AddDocCommentNodes = PredefinedCodeFixProviderNames.AddDocCommentNodes;
```

Comment:
💡 It is preferable to define these as non-constant pass-through properties.

```suggestion
        public string AddDocCommentNodes => PredefinedCodeFixProviderNames.AddDocCommentNodes;
```

This ensures Razor can still reference the original property name if the string changes.

Reply:
How does defining them as constants preclude them from referencing other property names?

Reply:
I am now going to assume that constants are inlined by the compiler. =)